npm/metrics/endpoints.go

```diff
@@ -0,0 +1,27 @@
+package metrics
+
+import "github.com/Azure/azure-container-networking/npm/util"
+
+func RecordListEndpointsLatency(timer *Timer) {
+	if util.IsWindowsDP() {
```

Comment:
could probably move all these checks to a `*_windows.go` file but this works too :)

Reply:
moved all new metrics to *_windows.go files